src/js/merkle-drop/api/web3.js

```diff
@@ -16,7 +10,7 @@ export async function claimTokens(
   )
```

Comment:
Somewhere here we should check if we are on the right chain, and have selected the right account. 
If not, we show an error message.